src/Worker/UpdateServerPeers.php

```diff
@@ -57,13 +57,13 @@ public static function execute(string $url)
 		$total = 0;
 		$added = 0;
 		foreach ($peers as $peer) {
-			if (Network::isUrlBlocked('http://' . $peer)) {
+			if (Network::isUrlBlocked('https://' . $peer)) {
 				// Ignore blocked systems as soon as possible in the loop to avoid being slowed down by tar pits
 				continue;
 			}
 
 			++$total;
-			if (DBA::exists('gserver', ['nurl' => Strings::normaliseLink('http://' . $peer)])) {
+			if (DBA::exists('gserver', ['nurl' => Strings::normaliseLink('https://' . $peer)])) {
```

Comment:
This is ineffective since `Strings::normaliseLink` whole purpose is to reduce `https` URL to `http`.
```suggestion
			if (DBA::exists('gserver', ['nurl' => 'http://' . $peer])) {
```

Reply:
So `nurl` contains the URL without any SSL, just like an namespace identifier you may know from XML files?

Reply:
I'm not sure what you are referring to, but yes, normalized URLs are without SSL. We assume that the same content is served whether the same URL is accessed with or without SSL, so we normalize without SSL to prevent duplicate entries for contacts and servers.

Reply:
Okay, then I revert this part and take your suggestion.